test/e2e/proxy_test.go

```diff
@@ -51,6 +50,9 @@ func proxyEnabled(data *TestData) (bool, error) {
 }
 
 func TestProxyServiceSessionAffinity(t *testing.T) {
+	// Todo: add check for IPv6 address after Antrea Proxy is supported in IPv6
```

Comment:
Hi @ksamoray , currently I skip these e2e tests for Antrea Proxy in IPv6. After your changes for Service is ready, please remember to change the related to code to ensure the test cases could cover your code.